### ISSUE TITLE
[GUI] Editting a finded patient information clear my find result

### ISSUE THREAD

**Issue by nus-se-script** [nus-cs2103-AY2425S1/pe-dev-response#1527]
Step to reproduce:
1) `find john` will give something like this:

![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe/master/files/abe1b6ba-41bd-4acc-a7d9-783cd741c5aa.png)

2) Then, `edit 2 n/ Pete Does`

![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe/master/files/76467b1e-e032-40a1-a4c6-59a8698d9b02.png)

3) Then, press command for execution.

Expected output: The find list after step 1 remains with the updated name of the patient.

Actual output: The whole list of all patient return

![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2425S1/pe/master/files/084b12a5-41f8-43ea-951a-6c1acbe805b7.png)

As an user who is focusing on the list of found patients, I want the page to remain in order to operate on the returned records from find command. However, returning the entire list might be troublesome for the user to enter find command again to see the edited field. If in case I want to check if the patient from the find command is editted correctly, I have to scroll all the way through the entire list to check, which is really inconvenient. Hence, I would give this a medium severity.

<hr>
<sub>[original: nus-cs2103-AY2425S1/pe-interim#634] [original labels: severity.Medium type.FeatureFlaw]</sub>

**Comment by Johnwz123** [comment]
# Team's Response
No details provided by team.
## Duplicate status (if any):
Duplicate of #1934
